app/components/EntityView/index.js

```diff
@@ -7,42 +7,74 @@
 import React from 'react';
 // import styled from 'styled-components';
 
-import { FormattedMessage } from 'react-intl';
-import messages from './messages';
+import ViewWrapper from 'components/basic/ViewWrapper';
+import Grid from 'grid-styled';
+
+import Row from 'components/basic/Row';
+import ViewHeader from './ViewHeader';
+import ViewBody from './ViewBody';
+import ViewFooter from './ViewFooter';
+
 
 class EntityView extends React.PureComponent { // eslint-disable-line react/prefer-stateless-function
+  renderSection = (fields) => fields.map((field, index) => (
+    <div key={index}>
+      <p>{field.value}</p>
+    </div>
+  ))
   render() {
-    const { type, title, description, draft, updatedAt, targetDate } = this.props;
+    const { fields } = this.props;
     return (
       <span>
-        <h1>{type}</h1>
-        <FormattedMessage {...messages.title} />
-        <h3>{title}</h3>
-        <FormattedMessage {...messages.description} />
-        <p>{description}</p>
-        <FormattedMessage {...messages.draft} />
-        <p>{draft === false ? 'YES' : 'NO'}</p>
-        <FormattedMessage {...messages.updatedAt} />
-        <p>{updatedAt}</p>
-
-        {targetDate &&
-          <span>
-            <FormattedMessage {...messages.targetDate} />
-            <p>{targetDate}</p>
-          </span>
-        }
+        <ViewWrapper>
+          { fields.header &&
+            <ViewHeader>
+              <Row>
+                <Grid sm={3 / 4}>
+                  { fields.header.main &&
+                    this.renderSection(fields.header.main)
+                  }
+                </Grid>
+                <Grid sm={1 / 4}>
+                  { fields.header.aside &&
+                    this.renderSection(fields.header.aside)
+                  }
+                </Grid>
+              </Row>
+            </ViewHeader>
+          }
+          { fields.body &&
+            <ViewBody>
+              <Row>
+                <Grid sm={3 / 4}>
+                  { fields.body.main &&
+                    this.renderSection(fields.body.main)
+                  }
+                </Grid>
+              </Row>
```

Comment:
@tony-luisi closing (and re-opening) the row here will make the whole Grid thing useless